test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/Package/DeprecationCapabilityTests.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using NuGet.Versioning;
+using NuGet.VisualStudio.Internal.Contracts;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test
+{
+    public class DeprecationCapabilityTests
+    {
+        [Fact]
+        public void IsDeprecated_ReturnsExpected()
```

Comment:
Please name test methods according to naming convention in our coding guidelines.